scripts/firedrake-install

```diff
@@ -898,7 +901,8 @@ def build_and_install_libsupermesh():
                             "-DCMAKE_INSTALL_PREFIX=" + firedrake_env,
                             "-DMPI_C_COMPILER=" + cc,
                             "-DMPI_CXX_COMPILER=" + cxx,
-                            "-DMPI_Fortran_COMPILER=" + f90])
+                            "-DMPI_Fortran_COMPILER=" + f90,
+                            "-DCMAKE_Fortran_COMPILER" + f90])
```

Comment:
Old (but still installed on LTS Ubuntu, HPC etc...) versions of CMake fail to detect the right FORTRAN compiler, this is an issue for instance when using the Intel compiler, which requires a GCC compiler on path for C++11 support required elsewhere. CMake will pick up the GCC FORTRAN and then fail to build.

Is there any harm with manually specifying the FORTRAN compiler here?